kivy/uix/scrollview.py

```diff
@@ -798,14 +798,24 @@ def on_scroll_start(self, touch, check_children=True):
             return self.simulate_touch_down(touch)
 
         if in_bar:
+            e = None
```

Comment:
`e` means what exactly?

Reply:
Look at the entire file `e` is in this file used to hold temporary value of effect_x and effect_y in scrollview. Following existing usage.